src/utils/workspace.ts

```diff
@@ -32,3 +35,15 @@ export function isFolderOpenInWorkspace(fsPath: string): boolean {
         return false;
     }
 }
+
+export function getProjectType(projectPath: string): string {
+    let language: string = TemplateLanguage.JavaScript;
+    fse.readdirSync(projectPath).forEach((file: string) => {
+        const stat: fse.Stats = fse.statSync(path.join(projectPath, file));
+        // Currently checking the existing pom.xml to determine whether the function project is Java language based.
```

Comment:
Isn't pom.xml at the root of the project? Couldn't you just do `fse.pathExists(path.join(projectPath, 'pom.xml'))`?

If it's not always at the root, then we need to find a different way to detect Java projects - iterating through an entire folder has bad performance for the user

Reply:
Yes it's at the root path. I like the idea of using ```fse.pathExists()```